app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -2474,3 +2453,27 @@ msgstr "o'r meini prawf asesu"
 
 #~ msgid "Apply for funding to save an asset in your community"
 #~ msgstr "Gwneud cais am gyllid i achub ased yn eich cymuned"
+
+#~ msgid ""
+#~ "Start or continue an application for "
+#~ "funding to save an asset in your"
+#~ " community"
+#~ msgstr "Gwneud cais am gyllid i achub ased yn eich cymuned"
+
+#~ msgid ""
+#~ "You must have received an invitation "
+#~ "to apply. If we did not invite "
+#~ "you, first "
+#~ msgstr ""
+#~ "Mae'n rhaid i chi fod wedi derbyn"
+#~ " gwahoddiad i ymgeisio. Os na "
+#~ "wnaethom eich gwahodd, "
+
+#~ msgid "express your interest in the fund"
+#~ msgstr "mynegwch eich diddordeb yn y gronfa yn gyntaf"
+
+#~ msgid "Opening times:"
+#~ msgstr "Amserau agor:"
+
+#~ msgid "Closed:"
+#~ msgstr ""
```

Comment:
is this section meant to be commented out?

Reply:
Yes - these are auto generated and it means last time we updated translations these were needed, but now they are not. Next time we run update if they still aren't there they will be removed.

Reply:
ah okay - good to know 👍